src/components/SpecialResolution/ResolutionEditor.vue

```diff
@@ -147,12 +164,17 @@ export default class ResolutionEditor extends Vue {
   @Action(useStore) setSpecialResolution!: ActionBindingIF
   @Action(useStore) setSpecialResolutionValid!: ActionBindingIF
 
+  @Prop({ default: false }) readonly isEditing!: boolean
+
   $refs!: {
     resolutionDatePickerRef: DatePickerShared,
   }
 
-  resolutionDateText = ''
   resolution = ''
+  resolutionDateText = ''
+  // Used for undo for corrections.
+  resolutionOriginal = ''
```

Comment:
I'd prefer to see the comment on the line it applies to. Or you can use JSDoc format.

This isn't a big deal but it's inconsistent with other property comments and I wondered whether it applied to line 176 or line 175 (and missing a CR).